src/ShaderHelper.h

```diff
@@ -49,13 +49,6 @@ typedef struct {
 

```

Comment:
Sounds good, was thinking about that but didn't know if you wanted it in the ShaderHelper for a specific reason.